ui/src/plugins/epl-ets/components/TableView.js

```diff
@@ -0,0 +1,285 @@
+    }
+  }
+
+  render() {
+    var tabData = this.props.table.tabs.filter(function(node) {
+      var newName = this.split('.').join('_').replace('@', '_at_');
+      return node.name === newName;
+    }, this.props.table.node);
+    if (tabData[0].length < 1) return null;
+    const list = tabData[0].tabs.map(function({ info, call_stats, ...a }) {
+      var callStatsLookup = {
+        lookupMax: 0,
+        lookupCount: 0
+      };
+      var callStatsInsert = {
+        insertMax: 0,
+        insertCount: 0
+      };
+      if (call_stats !== 'undefined' && typeof call_stats !== 'undefined') {
+        var lookup = call_stats.filter(function(obj) {
+          return obj.func === 'lookup';
+        });
+        var insert = call_stats.filter(function(obj) {
+          return obj.func === 'insert';
+        });
+        if (lookup.length !== 0) {
+          callStatsLookup.lookupMax = lookup[0].max_time;
+          callStatsLookup.lookupCount = lookup[0].count;
+        }
+        if (insert.length !== 0) {
+          callStatsInsert.insertMax = insert[0].max_time;
+          callStatsInsert.insertCount = insert[0].count;
+        }
+      }
+      var callStatsObj = {
+        ...callStatsInsert,
+        ...callStatsLookup
+      };
+      return { ...a, ...info, ...callStatsObj };
+    });
+
+    const listSorted = this.state.sortBy ? this.sort(list) : list;
+    const rowGetter = ({ index }) => listSorted[index];
+    return (
+      <AutoSizer>
+        {({ width, height }) =>
+          <div>
+            <div>
+              <h4 className="selection">
+                {this.state.selectLabel}
+              </h4>
+              <Select
+                name="size-select"
+                multi={true}
+                onChange={selected => {
+                  this.setState({ selectedOptions: selected });
+                }}
+                options={this.state.selectOptions}
+                value={this.state.selectedOptions}
+                autosize={true}
+                className={'ets-table-select'}
+                optionClassName={'ets-table-select-opts'}
+              />
+            </div>
+            <Table
+              ref="Table"
+              disableHeader={false}
+              width={width}
+              headerHeight={60}
+              height={height - 160}
+              rowHeight={30}
+              rowGetter={rowGetter}
+              rowCount={list.length}
+              sortBy={this.state.sortBy}
+              sortDirection={this.state.sortDirection}
+              rowClassName={({ index }) => {
+                if (index !== -1) {
+                  return 'ets-table-row';
+                } else {
+                  return 'ets-table-header';
+                }
+              }}
+              sort={({ sortBy, sortDirection }) => {
+                this.setState({ sortBy, sortDirection });
+              }}
+            >
+              {this.is_selected('name')
```

Comment:
In cases like that instead of ternary operator you can write something like this:
```js
{this.isSelected('name') && <Component />}
```
It's smaller and output JSX will be the same